packages/plugins/minos-discovery-kong/README.md

```diff
@@ -27,7 +27,6 @@ Modify `config.yml` file:
 ```yaml
 ...
 discovery:
-  connector: minos.networks.DiscoveryConnector
```

Comment:
Why did you remove this line? I think with `minos.common.ConfigV2` is a mandatory field.

Reply:
Done!